qutebrowser/commands/command.py

```diff
@@ -173,6 +176,9 @@ def _check_prerequisites(self, win_id):
                 "{}: Only available with {} "
                 "backend.".format(self.name, self.backend.name))
 
+        if count == 0:
```

Comment:
The missing bit here is that we actually want to allow a 0 count when the argument has `zero_count=True` set, but we don't actually have the necessary information here.

I'd propose something like this (untested):

- In `Command.__init__`, set `self._zero_count_allowed = None` (`None` because we don't know yet), and describe the argument in the docstring
- Here, do `if count == 0 and not self._zero_count_allowed:`
- Now we need to find out which argument is the count, and then set `self._zero_count_allowed` according to that - the best place to do that is probably in `_inspect_func`, in the for loop running for every parameter. There you can get the `ArgInfo` object using `self._get_arg_info(param)`, and if `arg_info.count` is true (i.e. we're looking at the `count` argument), set `self._zero_count_allowed = arg_info.zero_count`

I hope that made sense? If not, feel free to ask! :smile: